### DISCUSSION TITLE
Smart categories and product relation modules?

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#3944] [Extensions & Modules]
Hi everybody,
I'm looking for 2 modules
- smart categories (create categories that gets products associated via automatic rules)
- rule based product relation (create upsell/crosssell/related relations with rules instead of manually)

is there something out there in 2024?

**Comment by Axepih** [comment]
This https://www.wyomind.com/dynamic-category-magento.html ?

**Reply by fballiano** [reply]
thanks!

**Comment by empiricompany** [comment]
@fballiano i use this two extensions:

>     * smart categories (create categories that gets products associated via automatic rules)

https://github.com/firegento/firegento-dynamiccategory (we can add it to our awesome list too)

>     * rule based product relation (create upsell/crosssell/related relations with rules instead of manually)

https://aheadworks.com/automatic-related-products-extension-for-magento-1

**Reply by fballiano** [reply]
thanks! great modules!

some comments on top of my head:
- aheadworks, did they take the screenshots of the magento1 frontend in 1965? how old is that freaking theme? can't they update it already?
- firegento, the module is not on packagist, I'll try contacting some other admins that developed it to see if they can add it, also... it's GPL... so I'll talk about that too

**Comment by empiricompany** [comment]
if you want only to populate categories by newer and/or on sale products:
https://github.com/empiricompany/magento-autocategories

**Reply by fballiano** [reply]
thanks man!

**Comment by Curious-Aria** [comment]
We use the following dynamic category extension by magefan https://magefan.com/magento-2-dynamic-categories
the other vendors crashed with our large catalog

**Reply by addison74** [reply]
That extension is for Magento 2.

**Reply by Curious-Aria** [reply]
yes
